src/Weikio.ApiFramework.Plugins.AzureAD/Users/UserApi.cs

```diff
@@ -125,7 +125,7 @@ public async Task<ActionResult<UserDto>> UpdateUserDepartment(string user, strin
             }
         }
 
-        public async Task<ActionResult<UserDto>> CreateInvititation(string email, bool sendInvitationMessage, string inviteRedirectUrl)
+        public async Task<ActionResult<UserDto>> CreateInvititation(string email, bool sendInvitationMessage, string inviteRedirectUrl, string invitationLanguage)
```

Comment:
I suppose at some point we should introduce parameter object. But that is a breaking change which requires new major version.

Reply:
Yeah, no point of expanding the parameter list too much. I would say we're right on the edge.